formal-spec/measurements.tex

```diff
@@ -156,7 +159,7 @@ \section{Impact on performance}
 \end{figure}
 
 We can see that the impact on the system's performance is negligible even when
-we consider $100,000$ participants.
+we consider $100,000$ participants. Moreover, we see that the usage consumption percentage scales linearly in the number of participants, i.e., a 10 times increase in the number of participants will only increase 10 times the required usage percent. Also, if we double the throughput, then we can process a double amount of participants workload at the same time.
```

Comment:
Limit the line length to 80 characters. It's easier to review :)

Reply:
OK